src/internal/packager/git/url.go

```diff
@@ -15,7 +15,7 @@ import (
 // For further explanation: https://regex101.com/r/zq64q4/1
 var gitURLRegex = regexp.MustCompile(`^(?P<proto>[a-z]+:\/\/)(?P<hostPath>.+?)\/(?P<repo>[\w\-\.]+?)(?P<git>\.git)?(?P<atRef>@(?P<ref>[\w\-\.]+))?$`)
 
-// MutateGitURlsInText Changes the giturl hostname to use the repository Zarf is configured to use
+// MutateGitUrlsInText changes the giturl hostname to use the repository Zarf is configured to use.
```

Comment:
Shouldn't this be `MutateGitURLsInText`

Reply:
yeah I didnt make every correction, but welcome more, there's about 90 of them :-D

Reply:
Fixed :wink: